src/Microsoft.Sbom.Api/SBOMValidator.cs

```diff
@@ -77,12 +89,18 @@ public async Task<bool> ValidateSbomAsync()
 
         inputConfig.ToConfiguration();
 
+        var sbomConfig = sbomConfigs.Get(configuration.ManifestInfo.Value.FirstOrDefault());
```

Comment:
You should be able to use inputConfig.ManifestPath(). This would remove the need for an ISbomConfigProvider in the constructor.

Reply:
When I try to use this, I get a build error saying that ManifestPath is obsolete.

Reply:
Using inputConfig.ManifestDirPath() instead should work. This will give you the path to the _manifest directory under which the SBOM will always be found.

Reply:
But it only gives you the directory, right? Not the full path to the actual manifest file? In the case that the directory exists but the file doesn't exist, I think we would still want to throw an error.

Reply:
You're right, even though this should never be the case. In theory it is possible for the _manifest to be present the spdx not be present. So we can be safe and check for it directly using the sbomConfig in the new constructor.